lib/sign.ts

```diff
@@ -11,15 +11,18 @@ const sign = (xml: string, signingKey: string, publicKey: string, xPath: string)
     throw new Error('Please specify signingKey');
   }
 
-  const sig = new xmlcrypto.SignedXml();
+  const sig = new SignedXml();
```

Comment:
`privateKey`,`getKeyInfoContent`, `signatureAlgorithm` can all be part of SignedXml constructor options.

Reply:
Can you make the changes please.